packages/farcaster/src/channels.ts

```diff
@@ -1,8 +1,51 @@
-export type Channel = {
+// old, deprecated
+export type Channelv1 = {
   // null represents the default, "Home" channel
   channel_id: string | null;
   parent_url: string | null;
   name: string;
   image: string | null;
   description?: string | null;
 };
+
+export type Channel = VirtualChannel | RealizedChannel;
+
+export const homeVirtualChannel = {
+  id: "",
+  description: "followers",
+  name: "Home",
+  object: "channel",
+  parent_url: null,
+  image_url: "https://warpcast.com/~/channel-images/home.png",
+  channel_id: "home",
+} as const;
+
+export type VirtualChannel = typeof homeVirtualChannel;
```

Comment:
I think a generic `VirtualChannel` type would be useful for a reusable lighter representation of a channel e.g. for displaying in a list